elastic/apm-server
# Otel span mapping

## Motivation/summary

translate incoming OTel attributes from agents into the expected apm transaction/span values using the same mapping applied in otlp intake

## Checklist


## How to test these changes

Ingest transactions/spans with an `"otel": {"span_kind": "kind", "attributes": { // insert common attributes here } }` kv pair in the json. Verify that the values in `attributes` have been correctly translated to their expected fields in the spans and transactions.

## Related issues

requested in https://github.com/elastic/apm-server/issues/6259
